psi4/src/psi4/scfgrad/scf_grad.cc

```diff
@@ -685,6 +686,307 @@ SharedMatrix SCFDeriv::compute_hessian()
+                            // CyBz
+                            ECPp[Cy][Bz] += tmpBzCy;
+                            // CzBx
+                            ECPp[Cz][Bx] += tmpBxCz;
+                            // CzBy
+                            ECPp[Cz][By] += tmpByCz;
+                            // CzBz
+                            ECPp[Cz][Bz] += BCscale*tmpBzCz;
+                            // CxCx
+                            ECPp[Cx][Cx] += tmpCxCx;
+                            // CyCy
+                            ECPp[Cy][Cy] += tmpCyCy;
+                            // CzCz
+                            ECPp[Cz][Cz] += tmpCzCz;
+                            // CxCy
+                            ECPp[Cx][Cy] += tmpCxCy;
+                            // CxCz
+                            ECPp[Cx][Cz] += tmpCxCz;
+                            // CyCz
+                            ECPp[Cy][Cz] += tmpCyCz;
+
+                            ++AxAx;
+                            ++AxAy;
+                            ++AxAz;
+                            ++AyAy;
+                            ++AyAz;
+                            ++AzAz;
+                            ++AxBx;
+                            ++AxBy;
+                            ++AxBz;
+                            ++AyBx;
+                            ++AyBy;
+                            ++AyBz;
+                            ++AzBx;
+                            ++AzBy;
+                            ++AzBz;
+                            ++AxCx;
+                            ++AxCy;
+                            ++AxCz;
+                            ++AyCx;
+                            ++AyCy;
+                            ++AyCz;
+                            ++AzCx;
+                            ++AzCy;
+                            ++AzCz;
+                            ++BxBx;
+                            ++BxBy;
+                            ++BxBz;
+                            ++ByBy;
+                            ++ByBz;
+                            ++BzBz;
+                            ++BxCx;
+                            ++BxCy;
+                            ++BxCz;
+                            ++ByCx;
+                            ++ByCy;
+                            ++ByCz;
+                            ++BzCx;
+                            ++BzCy;
+                            ++BzCz;
+                            ++CxCx;
+                            ++CxCy;
+                            ++CxCz;
+                            ++CyCy;
+                            ++CyCz;
+                            ++CzCz;
+                        }
+                    }
+                }
+            }
+        }
+        // Symmetrize the result
+        int dim = hessians_["ECP"]->rowdim();
+        for (int row = 0; row < dim; ++row){
+            for (int col = 0; col < row; ++col){
+                ECPp[row][col] = ECPp[col][row] = (ECPp[row][col] + ECPp[col][row]);
+            }
+        }
```

Comment:
Tested after rebase against #2368. Label change heals general Hessian segfault. Commenting heals tests. When uncommented, material errors remain in Hessian terms (e.g., scf-hess1). I think I've met complications in symmetrizing the Hessian before -- had to do a block symmetrize or something in Py.
```suggestion
        int dim = hessians_["Effective Core Potential"]->rowdim();
        //for (int row = 0; row < dim; ++row){
        //    for (int col = 0; col < row; ++col){
        //        ECPp[row][col] = ECPp[col][row] = (ECPp[row][col] + ECPp[col][row]);
        //    }
        //}
```